frontend/src/index.scss

```diff
@@ -0,0 +1,2 @@
+$black: black;
```

Comment:
Tip: you should name your variables with their function, not their content (`$main` instead of `$black` for example). 
Else, if the client wishes to change to purple main color, you'll store the "purple" value in the `$black` variable, and I'll laugh. Lots and lots. And you won't enjoy this :p